types/kotlin2/MobileScreen.kt

```diff
@@ -16,23 +16,28 @@
 
 package im.vector.app.features.analytics.plan
 
-import im.vector.app.features.analytics.itf.VectorAnalyticsScreen
+import im.vector.app.features.analytics.itf.VectorAnalyticsEvent
```

Comment:
The python generator needs to be updated to fix this.

Reply:
Good spot, thanks